lib/utils.js

```diff
@@ -212,7 +212,7 @@ var compact = function compact(value) {
 };
 
 var isRegExp = function isRegExp(obj) {
-    return Object.prototype.toString.call(obj) === '[object RegExp]';
+    return obj instanceof RegExp;
```

Comment:
unfortunately this is not short, it's broken. `instanceof` doesn't work across realms, and can be easily forged.